src/core/geometry/fuzzyfactory/FuzzyFactory3.js

```diff
@@ -26,15 +28,15 @@ FuzzyShape3Factory.prototype = {
   },
 
   getVertex: function (sourcevertex) {
-    let elements = [sourcevertex.pos._x, sourcevertex.pos._y, sourcevertex.pos._z]
+    let elements = [sourcevertex[0], sourcevertex[1], sourcevertex[2]]
     let result = this.vertexfactory.lookupOrCreate(elements, function (els) {
       return sourcevertex
     })
     return result
   },
 
   getPlane: function (sourceplane) {
-    let elements = [sourceplane.normal._x, sourceplane.normal._y, sourceplane.normal._z, sourceplane.w]
+    let elements = [sourceplane[0], sourceplane[1], sourceplane[2], sourceplane[3]]
```

Comment:
Yeah. It would be nice. I also thought of adding poly3.x() functions as well. But once you know that everything is stored as arrays in V2, then indexes 0, 1, 2, 3 just become natural.

Reply:
What I am suggesting is array notation, just replacing the magic numbers with meaningful constants.

Reply:
For the first three, maybe -- but not so sure about W ... :)